ai-server/main.py

```
@@ -146,4 +233,69 @@ async def upload_document(
+[질문]
+{request.question}
+
+[답변]
+"""
+
+    # 4. EXAONE LLM 호출. OllamaLLM.invoke()는 동기 함수이므로 await 없이 호출
+    answer = llm.invoke(prompt)
+
+    # 5. 출처 목록 구성: document_id, source(파일명), 청크 미리보기, 헤더 메타데이터 포함
+    sources = []
+    for doc, meta in zip(docs, metadatas):
```

Ruff (0.15.10)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)